### PR TITLE
Test Elixir standard library modules

### PR DESCRIPTION
**Pull request by bartblast**

### CODE REVIEW

**Comment by bartblast** [comment]
If you think this won't be helpful, just close it. No hard feelings! ;)

File: test/beam_file_test.exs [hunk 1]
```diff
@@ -360,6 +360,25 @@ defmodule BeamFileTest do
...
+
+      Enum.each(modules, fn module ->
+        try do
+          assert {:ok, _code} = BeamFile.elixir_code(module)
+        rescue
+          error ->
+            reraise %{error | file: to_string(module)}, __STACKTRACE__
+          end
+      end)
+    end
```
Comment (NickNeck) [inline test/beam_file_test.exs:381]: This is a great idea. I didn't know the function `Application.spec/2` at all.
I would rewrite the function a little bit.
```suggestion
    :elixir
    |> Application.spec(:modules)
    |> Enum.each(fn module ->
      unless module |> to_string |> String.starts_with?("elixir_") do
        test "returns elixir code for #{inspect(module)}" do
          assert {:ok, _code} = BeamFile.elixir_code(unquote(module))
        end
      end
    end)
```
With this change we get a test per module. I assume that it is not possible to run `elixir_code/1` with the `elixir_*` modules. But I will also take a look on this modules.

Comment (bartblast) [reply test/beam_file_test.exs]: one small problem: there is also :elixir module we need to filter out which won't be caught with elixir_*

Comment (NickNeck) [reply test/beam_file_test.exs]: Good point. `String.starts_with?("elixir")` should work.

**Pull request merged**